src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -548,8 +545,8 @@ class NetworkOPsImp final
 
     std::atomic<OperatingMode> mMode;
 
-    std::atomic <bool> mNeedNetworkLedger;
-    bool m_amendmentBlocked;
+    std::atomic <bool> needNetworkLedger_ {false};
+    std::atomic <bool> amendmentBlocked_ {false};
```

Comment:
Could we use `std::atomic_flag` for these? More importantly, _should_ we? @HowardHinnant, any guidance?